### DISCUSSION TITLE
Underscore in interactive windows reported as "not defined"

### DISCUSSION THREAD

**Discussion by hmijail** [microsoft/pylance-release#4732] [Enhancement]
## Environment data

-   Language Server version: v2023.8.30 (in Pylance extension)
-   OS and version: macOS Ventura 13.5 
-   Python version (& distribution if applicable, e.g. Anaconda): python 3.11.4 (brew)

## Code Snippet

<img width="697" alt="image" src="https://github.com/microsoft/pylance-release/assets/4139546/e911441c-fcd3-4f44-b783-602730d6e267">

This happens both in interactive windows and notebook windows, and with all of "_", "__", "_1", etc.

## Expected behavior

"_" and friends work as expected in interactive windows, so Pylance shouldn't complain that they are not defined.

**Comment by debonte** [comment]
These variables are documented here: http://ipython.org/ipython-doc/stable/interactive/reference.html#output-caching-system

**Comment by debonte** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
